framework/doc/packages_config.yml

```diff
@@ -6,3 +6,4 @@ minimum_python: 3.7
 mpich: 4.0.2
 petsc_alt: 3.11.4
 petsc: 3.16.6
```

Comment:
@cticenhour I should probably change this too!

Reply:
Yes - we should while we're here. Good catch!

Reply:
Thanks for all the fixups @milljm!